### ISSUE TITLE
Manifest merger failed at attribute allowBackup

### ISSUE THREAD

**Issue by ChristianBecker** [paypal/PayPal-Android-SDK#225]
Thank you for the release on Maven Central! Unfortunately I get an exception when trying to replace the local JAR with the Maven artifact:

```
Error:Execution failed for task ':processDebugManifest'.
> Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:41:9-36
  	is also present at [com.paypal.sdk:paypal-android-sdk:2.12.3] AndroidManifest.xml:23:18-44 value=(true).
  	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:39:5-173:19 to override.
```

As a workaround I can simply modify my `AndroidManifest.xml` to override the `allowBackup` property, but a proper fix would be the removal of the `allowBackup` property from your `AndroidManifest.xml`.

**Comment by jaypatel512** [comment]
You are right @ChristianBecker !

Removing the allowBackup will be the right solution. We are working on it right away. We will make a new release soon.

**Comment by jaypatel512** [comment]
Fixed in [2.12.4](https://github.com/paypal/PayPal-Android-SDK/releases/tag/2.12.4) release.
